src/sys/socket/addr.rs

```diff
@@ -702,7 +702,7 @@ pub mod sys_control {
             let addr = sockaddr_ctl {
                 sc_len: mem::size_of::<sockaddr_ctl>() as c_uchar,
                 sc_family: AddressFamily::System as c_uchar,
-                ss_sysaddr: consts::AF_SYS_CONTROL as uint16_t,
```

Comment:
Can you file a PR with upstream then? Seems like `AF_SYS_CONTROL` should be a `u16`, do you agree?